### ISSUE TITLE
Add support for CD

### ISSUE THREAD

**Issue by georgiy-belyanin** [NikitosKey/alcounting_bot#12]
It may be handy to test the bot on the flight by adding support for CD.

It would be nice to access the bot based on the main branch using @alcounting_bot telegram account.

**Comment by NikitosKey** [comment]
Yes, this feature will implement after adding [party management system](https://github.com/NikitosKey/alcounting_bot#party-management-system--checklist)
But you can already try actual version using this account.
